include/grpc++/support/sync_stream.h

```diff
@@ -151,17 +165,21 @@ class ClientReader GRPC_FINAL : public ClientReaderInterface<R> {
   Call call_;
 };
 
+/// Client-side interface for streaming writes of message of type \a W.
 template <class W>
 class ClientWriterInterface : public ClientStreamingInterface,
                               public WriterInterface<W> {
  public:
+  /// Block until writes are completed.
```

Comment:
Looking at https://github.com/grpc/grpc/blob/master/include/grpc%2B%2B/support/sync_stream.h#L187, wouldn't the call to Pluck block if the cq is empty?